docs/data/base/components/input/input.md

```diff
@@ -0,0 +1,58 @@
+---
+product: base
+title: React Input unstyled component and hook
+components: InputUnstyled
+githubLabel: 'component: input'
+packageName: '@mui/base'
+---
+
+# Unstyled input
+
+<p class="description">The `Input` component provides users with a field to enter and edit text.</p>
+
+Features:
+
+- ✨ Supports start and end adornments.
+- 🚀 Can be transformed to textarea using the `multiline` prop.
+- ♿️ Adds the appropriate ARIA roles automatically.
+
+## Basic input
+
+```js
+import InputUnstyled from '@mui/base/InputUnstyled';
+```
+
+{{"demo": "UnstyledInputBasic.js", "defaultCodeOpen": false}}
+
+## Adornments
+
+You can use the `startAdornment` and `endAdornment` props to add a prefix, suffix, or an action to an input.
+Common use cases of adornments include:
+
+- when an input receives a specific unit of measure (like kilograms or currency)
+- when an icon button toggles hiding/showing a password
+
+{{"demo": "InputAdornments.js", "defaultCodeOpen": false}}
+
+## Multiline
+
+The `multiline` prop transforms the `input` field into a `textarea` element.
```

Comment:
I don't know if we have a rule for this, but personally, I like to see HTML elements written like actual tags just to make it as clear as possible what we're discussing—especially when props are also mentioned.

```suggestion
The `multiline` prop transforms the `<input>` field into a `<textarea>` element.
```

Reply:
I don't think we are consistent in this, but we may start with this practice. I like it more too 👍